hack/make/test-docker-py

```diff
@@ -22,6 +22,9 @@ if [ -n "$TEST_INTEGRATION_USE_SNAPSHOTTER" ]; then
 	PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/api_build_test.py::BuildTest::test_build_squash"
 fi
 
+# TODO(thaJeztah) re-enable after https://github.com/docker/docker-py/pull/3336 is in the DOCKER_PY_COMMIT release.
+PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/api_build_test.pyy::BuildTest::test_build_container_with_target"
```

Comment:
Arf; 

```suggestion
PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/api_build_test.py::BuildTest::test_build_container_with_target"
```

Reply:
Maybe instead of skipping, let's just pull docker-by by commit? https://github.com/moby/moby/blob/04d381716adcc534505c63ebde93b5840804f70c/hack/make/test-docker-py#L8

Reply:
Yeah, there's a bunch of other tests that are also waiting for a new release; didn't want to include all of those in this PR (and ideally, look if we can have a new release of docker-py); let me keep that for a follow-up.

Mostly dusted-off this PR because we kept on forgetting about it 😂

Reply:
Either way, fixed the typo for now 👍